src/routes/expenseRoute.js

```diff
@@ -0,0 +1,12 @@
+const express = require('express');
+const expenseController = require('../controllers/expenseController');
+
+const expenseRoute = express.Router();
+
+expenseRoute.get('/', expenseController.getExpenses);
+expenseRoute.get('/:id', expenseController.getOneExpense);
+expenseRoute.post('/', expenseController.createExpense);
+expenseRoute.patch('/:id', expenseController.updateExpense);
+expenseRoute.delete('/:id', expenseController.deletExpense);
```

Comment:
There is a typo in the function name `deletExpense`. It should be corrected to `deleteExpense` to match the function name in the `expenseController` and ensure the DELETE route works correctly.